tinkerbell/stack/values.yaml

```diff
@@ -43,7 +43,8 @@ boots:
   image: quay.io/tinkerbell/boots:v0.8.0
   tinkWorkerImage: quay.io/tinkerbell/tink-worker:v0.8.0
   trustedProxies: []
-  remoteIp: 192.168.1.94
+  # In most cases this value will be the same as `stack.loadBalancerIP`.
+  remoteIp: 192.168.2.111
```

Comment:
Do we want to make this empty and then require it?

Reply:
hmmm....dont know if i know what that would look like?

Reply:
oh....so `remoteIp: ""` and then use a require in the boots chart?

Reply:
is `remoteIp` optional in the Boots chart? cant it be overridden in such a way that it becomes optional?

Reply:
hmmm....i think i see. Let me see about adding a require in the Boots chart.

Reply:
Yes, just in the boots chart should we require it so there's a clear error when unspecified. 

Thinking about it more, it is a little nuanced because you could override all the things that inherit from it at which point its redundant. Options: (1) Add required and the Helm logic to deal with the nuances; (2) Add required and accept it's going to bork with a full set of overrides; (3) Just use a comment.

Reply:
hmm...looked into added the required, but it doesnt feel right as `remoteIp` isnt necessarily always required. I think a comment is the way to go, for now at least. maybe once we ppl start using it we find one of the other options makes more sense.